app/models/concerns/crosscitable.rb

```diff
@@ -103,7 +103,7 @@ def xml=(value)
 
       @schema_version = bolognese.schema_version || "http://datacite.org/schema/kernel-4"
       @from = bolognese.from
-      @raw = bolognese.raw
+      @raw = value
```

Comment:
This change doesn't work if `value` is metadata in another format, and not DataCite XML.

Reply:
ok; So we are clear on that the defualt storage format is always DataCite XML. Good. Then, bolognese.raw is not working when the original format is datacite-xml

Reply:
> bolognese.raw is not working when the original format is datacite-xml

Why is that?

Reply:
I was thinking that this might be true because the spec (file below) was always failing in the comparison of XML. however, upon further checking, it seems that Bolognese (something in LUPO) `prettify` XML (adding tabs) and the difference on XML check comes from there.  Therefore I think comparing XML on input vs XML on output is not correct (i'll check my spec) . I still think the getter (file above) is the main issue: there shouldn't be any conversion. I'll focus on that.